octodns/processor/arpa.py

```diff
@@ -79,12 +85,13 @@ def populate(self, zone, target=False, lenient=False):
         for arpa, fqdns in self._records.items():
             if arpa.endswith(f'.{zone_name}'):
                 name = arpa[:-n]
+                _, record_ttl, _ = fqdns[0]
                 # Note: this takes a list of (priority, fqdn) tuples and returns the ordered and uniqified list of fqdns.
```

Comment:
It appears there's an extra bit in the tuple being passed now so the comment/doc needs to be updated:

```suggestion
                # Note: this takes a list of (priority, ttl, fqdn) tuples and returns the ordered and uniqified list of fqdns.
```

Related to the `[0]` comment above perhaps it makes sense to have `_order_and_unique_fqdns` pick the ttl and return a tuple of `(ttl, fqdn)` so that it can use the ttl of the post ordered "first" occurance of each fqdn.

Relatedly we'll want some tests around the behavior to make sure it works as defined/expected and doesn't regress in the future.

Reply:
I'm not sure I understand this correctly, but it now takes the TTL value of the first ordered FQDN (the one with the lowest priority). I have also added a test for this.